app/components/footer.tsx

```diff
@@ -1,63 +1,29 @@
+'use client';
+import { PageFooter } from '@developmentseed/veda-ui';
```

Comment:
Move this import to this file [here](https://github.com/developmentseed/next-veda-ui/blob/main/app/lib/index.ts). File explains that it marks it with the directive here so you dont have to do it directly like so https://github.com/developmentseed/next-veda-ui/blob/main/app/lib/index.ts#L30-L33.

Reply:
Then you'd import it like `import { PageFooter } from '@lib';` here 👍🏼

Reply:
Interesting! Thanks for the explanation @sandrahoang686 and the implementation @snmln :+1: